ddtrace/contrib/botocore/patch.py

```diff
@@ -453,10 +484,37 @@ def patched_api_call(original_func, instance, args, kwargs):
         span.set_tag(ANALYTICS_SAMPLE_RATE_KEY, config.botocore.get_analytics_sample_rate())
 
         try:
-            result = original_func(*args, **kwargs)
-            _set_response_metadata_tags(span, result)
+            if endpoint_name == "sqs" and operation == "ReceiveMessage" and config._data_streams_enabled:
+                queue_url = params["QueueUrl"]
+                queue_name = queue_url[queue_url.rfind("/") + 1 :]
+
+                if "MessageAttributeNames" not in args[1]:
+                    args[1].update({"MessageAttributeNames": ["_datadog"]})
+                    args = (args[0], args[1])
+                elif "_datadog" not in args[1]["MessageAttributeNames"]:
+                    args[1].update({"MessageAttributeNames": args[1]["MessageAttributeNames"] + ["_datadog"]})
+                    args = (args[0], args[1])
```

Comment:
what if you have args[2]? it would break the args.

Reply:
the way it's structured is that args[0] will be "ReceiveMessage" and args[1] is the actual list of parameters, ie `('ReceiveMessage', {'QueueUrl': <<URL here>>, 'MaxNumberOfMessages': 3, 'WaitTimeSeconds': 2, 'AttributeNames': ['All']})`, so it doesn't break the args because `MessageAttributeNames` is added to args[1]